opm/parser/eclipse/EclipseState/Schedule/SummaryState.hpp

```diff
@@ -73,6 +75,10 @@ class SummaryState {
     */
     void set(const std::string& key, double value);
 
+    bool del(const std::string& key);
+    bool del_well_var(const std::string& well, const std::string& var);
+    bool del_group_var(const std::string& group, const std::string& var);
```

Comment:
Could these be named `erase` instead?  That would tie better into the convention of the standard library.

Reply:
Have changed to `erase`